jax/random.py

```diff
@@ -311,41 +313,78 @@ def shuffle(key, x, axis=0):
   return x
 
 
-@partial(jit, static_argnums=(1, 2))
-def normal(key, shape, dtype=onp.float32):
-  """Sample standard normal random values with given shape and float dtype.
+@partial(jit, static_argnums=(3, 4))
+def normal(key, loc=0.0, scale=1.0, shape=(), dtype=onp.float32):
+  """Sample normal random values with given shape, mean, variance and float dtype.
 
   Args:
     key: a PRNGKey used as the random key.
-    shape: a tuple of nonnegative integers representing the shape.
+    loc: optional, the mean of the distribution (default 0.0).
+    scale: optional, the standard deviation of the distribution (default 1.0).
```

Comment:
I think that we can keep the current behaviour (without `loc`, `scale`). Many samplers can be transformed by `x -> loc * x + scale` so it is unnecessary to add `loc`, `scale` for all of them.